examples/Training/java/src/training/RawDataAccess.java

```diff
@@ -141,7 +141,9 @@ private void retrieveTile()
             }
         } catch (Exception e) {
             throw new Exception("Cannot read the tiles", e);
-        } 
+        } finally {
+            rdf.close();
```

Comment:
@joshmoore @mtbc certainly something to look into
so we can simplify good portion of code